src/components/BurgerMenu/BurgerMenu.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './BurgerMenu.scss';
```

Comment:
Usually, in the project there is agreement to put styles imports under all other imports. It looks nicer. It's not a mistake but just 'good to have' thing.
and the order of imports is like
 --- 3d party libs import 
--- your components imports
 --- pictures and styles imports